api/v1alpha1/openapi.yaml

```diff
@@ -4527,6 +4532,70 @@ components:
       additionalProperties:
         type: string
         description: A user-defined custom property value.
+    SystemdUnitStatus:
+      type: object
+      required:
+        - unit
+        - load
+        - active
```

Comment:
my only input is some of this status is lower level, do we want to abstract any of this or do we feel we can educate the users about these details. they are fairly intuitive.

Reply:
I agree too.
I'm wondering if in the UI we should combine these into a handful of more directly understood statuses.